bin/Powheg/lheMacros/splitLheFiles.cpp

```diff
@@ -30,7 +30,11 @@ int main(int argc, char** argv)
   bool writeEvent = true;
   bool writePdf = false;
   int eventIt = 0;
+<<<<<<< HEAD
```

Comment:
in this case the "off-by-one" has been updated recently but, once more, I am not fully sure what to do